### PR TITLE
The BIG Merge

### PR DESCRIPTION
**Pull request by fernandonevarez**
Finished base layout of the website. In which layout has a Navbar that changes styles based off of current user's device and if the user is scrolling throughout the page. However with that each page of the website has a footer that allows the user to have access to important club links and each page also  has a custom header component that introduces the page and what kind of content can be found on the page.

### CODE REVIEW

File: src/components/page_layout/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+            <Link href={"/about#leadership"}>Leadership</Link>
+          </li>
+          <li className={desktopStyles.list_item}>
+            <Link
+              target="_blank"
+              href={
+                "https://asu.campuslabs.com/engage/organization/codedevils/"
+              }
+            >
+              SunDevilSnyc
```
Comment (reharri7) [inline src/components/page_layout/Footer.tsx]: Nit spelling error

File: src/components/page_layout/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            <Link target="_blank" href={"https://github.com/ASU-CodeDevils/"}>
+              Github
+            </Link>
+          </li>
+          <li className={desktopStyles.list_item}>
+            <Link
+              target="_blank"
+              href={"https://www.hackerrank.com/codedevils-spring-challenge"}
+            >
+              Hackerrank
```
Comment (reharri7) [inline src/components/page_layout/Footer.tsx]: Capitlization

**Comment by IMBCIT** [comment]
maxresdefault

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "java",
+      "name": "Launch Current File",
+      "request": "launch",
+      "mainClass": "${file}"
+    }
+  ]
+}
```
Comment (reharri7) [inline .vscode/launch.json:14]: New line

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+    "npm.packageManager": "pnpm"
+}
```
Comment (reharri7) [inline .vscode/settings.json:3]: new line

File: public/langs/spanish/data.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+const es = {
+  navbar: {
+    links: {
+      home: "Hogar",
+      about: "Acerca de",
+      contact: "Contacta con nosotras",
```
Comment (reharri7) [inline public/langs/spanish/data.ts:6]: Idk if you translated this but `nosotros`

File: src/server/routers/_app.ts [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+      text: z.string(),
+    })
+  ).query(({ input }) => {
+    return {
+      greeting: `Hello ${input.text}`,
+    };
+  }),
+});
+
+export type AppRouter = typeof appRouter;
```
Comment (reharri7) [inline src/server/routers/_app.ts:16]: new line

**Review by reharri7 — APPROVED** [review]
LGTM

**Pull request merged**